ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -100,7 +94,7 @@ ghcLibParserHsSrcDirs forParserDepends ghcFlavor lib =
         -- This next conditional just smooths over a 'master'
         -- vs. '8.8.1' branch difference (relating to a file
         -- 'GhcVersion.hs' IIRC).
-        [ stage0GhcBoot | ghcFlavor `elem` [ GhcMaster, Ghc8101, Ghc8102, Ghc901 ] ] ++
+        [ stage0GhcBoot | ghcFlavor `elem` [ GhcMaster, Ghc8101, Ghc8102, Ghc8103, Ghc901 ] ] ++
```

Comment:
This list seems to get longer and longer. Would it make sense to put an order on the `GhcXYZ`s and replace this with something like
```suggestion
        [ stage0GhcBoot | ghcFlavor >= Ghc8101 ] ++
```
Same question in multiple places below.

Reply:
Ok, https://github.com/digital-asset/ghc-lib/issues/265.